app/src/components/AtomComponents/CustomButton.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <button @click="handleClick" :class="buttonClass" :type="type">
+      {{caption}}
+    </button>
+  </template>
+  
+  <script>
+  export default {
+    props: {
+      buttonClass: String,
```

Comment:
```suggestion
      styleType: {
        type: String,
        default: 'primary',
        validator: value => ['primary', 'secondary'].includes(value)
      }
```
- `styleType` - will be more descriptive, I think
- use 'primary' and 'secondary' as css classes to style buttons --> 'primary' - default green button, 'secondary' - white button with green border